tests/Equinox.CosmosStore.Integration/DocumentStoreIntegration.fs

```diff
@@ -495,4 +495,37 @@ type Tests(testOutputHelper) =
         verifyRequestChargesMax 1
     }
 
+    [<AutoData(SkipIfRequestedViaEnvironmentVariable="EQUINOX_INTEGRATION_SKIP_COSMOS")>]
+    let ```Can safely evolve AccessStrategy over time`` cartContext = Async.RunSynchronously <| async {
+        let context = createPrimaryContext log 10
+        let unoptimized = Cart.createServiceWithoutOptimization log context
+        let snapshot = Cart.createServiceWithSnapshotStrategy log context
+        let rollingState = Cart.createServiceWithRollingState log context
+        let sku1 = SkuId(Guid.NewGuid())
+        let sku2 = SkuId(Guid.NewGuid())
+        let sku3 = SkuId(Guid.NewGuid())
```

Comment:
I left this in so you could tell me to make it a one liner and ignore my other styling faults.

Reply:
There's a Guid.gen, and if there isnt there should be a SkuId.gen ()
(Then we just need a Tuple.init3 and you have a most elegant one liner lol - I worked in a company that had a Marvel of a library where that absolutely would be added no questions asked)